TimeSeriesAnalysis.m

```diff
@@ -74,18 +85,18 @@
 %%############################################################################ 
 %% FIGURES: 
 %% Choose either figures or strength_figures (statistical strength), can't do both (bug due to octave update).
-%% 0 = no figures, 1 = show, 2 = save, 3 = show and save
-figures = 2; 
-strength_figures = 0;
+%% 0 = no figures, 1 = show, 2 = save, 3 = show and save (for both?)

```

Comment:
(for both?): Nej, du definerer det for både kørslen af de almindelige figurer (figures) og for kørslen af styrkeplottene (strength_figures). Men som nævnt, gør en opdatering i Octave, at mindst en af kørslerne skal defineres som 0, der kan altså ikke plottes figurer ud for begge i samme run.